### PR TITLE
you can now open airlocks and firedoors and stuff with a crowbar on combat mode by right clicking

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

what the title says

## Why It's Good For The Game

its so fucking bad when the station gets a tiny bit cold and now theres firelocks everywhere and you have to switch from combat mode to do anything with them

## Changelog
:cl:
qol: you can now open airlocks and firedoors and stuff with a crowbar on combat mode by right clicking
/:cl:

### CODE REVIEW

File: code/game/machinery/doors/door.dm [hunk 1]
```diff
@@ -221,7 +221,8 @@
 	return
 
 /obj/machinery/door/attackby(obj/item/I, mob/living/user, params)
-	if(!user.combat_mode && (I.tool_behaviour == TOOL_CROWBAR || istype(I, /obj/item/fireaxe)))
+	var/list/modifiers = params2list(click_parameters)
+	if((!user.combat_mode || LAZYACCESS(modifiers, RIGHT_CLICK)) && (I.tool_behaviour == TOOL_CROWBAR || istype(I, /obj/item/fireaxe)))
```
Comment (ATH1909) [inline code/game/machinery/doors/door.dm:225]:
```suggestion
	if((LAZYACCESS(modifiers, RIGHT_CLICK)) && (I.tool_behaviour == TOOL_CROWBAR || istype(I, /obj/item/fireaxe)))
```
let us do this while not in combat mode as well, please

File: code/game/machinery/doors/door.dm [hunk 2]
```diff
@@ -221,7 +221,8 @@
 	return
 
 /obj/machinery/door/attackby(obj/item/I, mob/living/user, params)
-	if(!user.combat_mode && (I.tool_behaviour == TOOL_CROWBAR || istype(I, /obj/item/fireaxe)))
+	var/list/modifiers = params2list(click_parameters)
```
Comment (Mothblocks) [inline code/game/machinery/doors/door.dm:224]: sir did you test this

image

Comment (Fikou) [reply code/game/machinery/doors/door.dm:224]: surely i did

File: code/game/machinery/doors/door.dm (hunk 1 above)
Comment (Fikou) [reply code/game/machinery/doors/door.dm:225]: lol

**Comment by Tlaltecuhtli** [comment]
can it just be right click with or without combat mode?

**Comment by Fikou** [comment]
> can it just be right click with or without combat mode?

the check is
if not combat mode: work always
if combat mode: work only if right click

**Review by Mothblocks — APPROVED** [review]

**Pull request merged**